packages/p2p-v2/src/components/FullPageMobileWrapper/FullPageMobileWrapper.scss

```diff
@@ -0,0 +1,39 @@
+.p2p-v2-mobile-wrapper {
+    @include mobile {
+        height: calc(100vh - 4rem);
+        width: 100vw;
+        margin: 0;
+        margin-top: 12rem;
+        border-radius: 0;
+        display: grid;
+        grid-template-rows: 6rem auto;
+
+        &--hide-deriv-app-header {
+            height: calc(100vh * 100);
```

Comment:
can we not just do 100vh? do we need to multiply by 100

Reply:
actually we don't need this thou, its from wallets